scripts/prox_crosswalk.py

```diff
@@ -34,7 +34,7 @@
 _YELP_ID_TO_PLACE_CACHE = {}
 
 def getAndCacheProviderIDs(keyID, providersList, identifiers):
-    providerIDs = _getCachedIDsForPlace(keyID, providersList)
+    providerIDs = {k: v for k,v in _getCachedIDsForPlace(keyID, providersList).items() if k in providersList}
```

Comment:
Is the motivation behind this addition to:

1. Ensure `k in providersList`?
1. Create a copy?

I believe 1 is taken care of inside `_getCachedIDsForPlace` and I don't think 2 is necessary.